lib/vanagon/platform/windows.rb

```diff
@@ -103,7 +104,7 @@ def generate_nuget_package(project)
       # @return [Array] list of commands required to build an msi package for the given project from a tarball
       def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
-        cg_name = "compfiles"
+        cg_name = "ProductComponentGroup"
```

Comment:
Just thinking aloud here, should this be a variable that is also used in the project template ? (see https://github.com/puppetlabs/vanagon/pull/271/files#r51787764). The downside is some extra complexity and that we would need to define the variable in the project.rb file.